### ISSUE TITLE
Proposal: Item removed event for UWP JumpList API

### ISSUE THREAD

**Issue by jaigak** [microsoft/WindowsAppSDK#169]
# Proposal: Item removed event for UWP JumpList API
Currently UWP JumpList API doesn't offer a removed event so the application can't track when user removes a jumplist item while the app is running.

## Summary
Some apps may use the jumplist to store certain things and currently it's not possible to know when something get's removed so apps can't update their UI.

## Scope

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to track jumplist item removal | Must |

**Comment by ShankarBUS** [comment]
Why? JumpList tasks can't be removed (I guess?) only Recent items can be removed. This is by design, correct. Why would any want to track the JumpList recent items?

Most of the users won't care about the JumpList recent items (if they do, they can pin it)

**Comment by Jaiganeshkumaran** [comment]
@ShankarBUS You can remove items which aren't in the tasks list.

**Comment by ShankarBUS** [comment]
@Jaiganeshkumaran,

> but you can create your own.

I know it. But I was confused on why you did want to track the removal.

So the OS just marks the items as removed and remove them from their UI. But the apps internal list will be the same, it should also be updated to prevent restore the removed item. Am I right?

**Comment by Jaiganeshkumaran** [comment]
@ShankarBUS Yes, some apps may use the jumplist as a storage for something and they can't know when it gets removed
